akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/ScreenshareModel.scala

```diff
@@ -88,6 +91,24 @@ object ScreenshareModel {
   def getHasAudio(status: ScreenshareModel): Boolean = {
     status.hasAudio
   }
+
+  def stop(outGW: OutMsgRouter, liveMeeting: LiveMeeting): Unit = {

```

Comment:
Would you consider keeping `stop` simply `this.resetDesktopSharingParams(liveMeeting.screenshareModel)` here but when calling it to also call the broadcast event?

Perhaps could even have `stop()` return boolean whether to broadcast an event? The benefit would be mostly keeping the model here clean

Reply:
This `stop` function was made this way to follow the pattern of `ExternalVideoModel`.